extensions/cms/package.nls.json

```diff
@@ -66,8 +68,17 @@
 	"cms.connectionOptions.commandTimeout.description": "The length of time (in seconds) to wait for a command to execute before terminating the attempt and generating an error",
 	"cms.connectionOptions.currentLanguage.displayName": "Current language",
 	"cms.connectionOptions.currentLanguage.description": "The SQL Server language record name",
-	"cms.connectionOptions.columnEncryptionSetting.displayName": "Column encryption",
-	"cms.connectionOptions.columnEncryptionSetting.description": "Default column encryption setting for all the commands on the connection",
+	"cms.connectionOptions.columnEncryptionSetting.displayName": "Always Encrypted",
+	"cms.connectionOptions.columnEncryptionSetting.description": "Enables or disables Always Encrypted for the connection",
+	"cms.connectionOptions.secureEnclaves.displayName": "Secure enclaves",
+	"cms.connectionOptions.secureEnclaves.description": "Enables or disables Secure enclaves for the connection",
```

Comment:
This is inconsistent casing of "Secure enclaves", most other places secure is not capitalized unless starting a phrase, so it appears to not be a proper noun name.  In any case, please review to ensure consistency.

Reply:
This was requested from AE team - so 'Secure enclaves' is as per their request.